test/unitTests/features/reportIssue.test.ts

```diff
@@ -37,19 +37,28 @@ suite(`${reportIssue.name}`, () => {
             version: "version2",
             isBuiltin: false
         },
-        id: "id2"
+        id: "id2",
+        extensionPath: "c:/extensions/xyz-x64"
+    };
+
+    const fakeDotnetInfo: DotnetInfo = {
+        FullInfo: "myDotnetInfo",
+        Version: "1.0.x",
+        RuntimeId: "1.1.x"
```

Comment:
Should this be a real fake value like `win10-x64`?

Reply:
For this particular test it doesn't make a difference, since report issue only cares about the FullInfo.  But I'll change it to something more real